docs/docs/behaviors/mouse-emulation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Mouse Emulation Behaviors
+sidebar_label: Mouse Emulation
+---
+
+## Summary
+
+Mouse emulation behaviors send mouse events. Currently, only mouse button presses are supported, but movement
+and scroll action support is planned for the future.
+
+## Configuration Option
+
+This feature can be enabled via a config option:
+
+```
+CONFIG_ZMK_MOUSE=y
+```
+
+If you use the mouse key press behavior in your keymap, the feature will automatically be enabled for you.
+
+## Mouse Button Defines
+
+To make it easier to encode the HID mouse button numeric values, include
+the [`dt-bindings/zmk/mouse.h`](https://github.com/zmkfirmware/zmk/blob/main/app/include/dt-bindings/zmk/mouse.h) header
+provided by ZMK near the top:
+
+```
+#include <dt-bindings/zmk/mouse.h>
+```
+
+The following defines are available:
+
+| Define        | Action         |
```

Comment:
I would put this table under "Behavior Binding" because they apply to `&mkp` behavior, and we can remove the "The following defines are available:" text in that case as well. What do you think?

Reply:
Moved.